src/coreComponents/linearAlgebra/DofManager.hpp

```diff
@@ -102,153 +111,93 @@ class DofManager
    * @param [in] connectivity Connectivity through what they are connected.
    * @param [in] regions names of regions where this coupling is defined.
-   */
-  void addCoupling( string const & rowFieldName,
-                    string const & colFieldName,
-                    Connector const connectivity,
-                    arrayView1d< string const > const & regions );
-
-  /**
-   * @brief Just another interface to allow four parameters (no regions, default is everywhere).
-   *
-   * @param [in] rowFieldName string the name of the row field.
-   * @param [in] colFieldName string the name of the col field.
-   * @param [in] connectivity Connectivity through what they are connected.
    * @param [in] symmetric bool is it symmetric, i.e., both row-col and col-row?
    */
   void addCoupling( string const & rowFieldName,
                     string const & colFieldName,
-                    Connector const connectivity,
-                    bool const symmetric );
+                    Connector connectivity,
+                    std::vector< string > const & regions = {},
+                    bool symmetric = true );
 
   /**
-   * @brief Add coupling between two fields.
-   * The connectivity argument defines how the two fields couple. If the first field has support location A,
-   * the second field has support location B, and the connecting object is C, the sparsity pattern will be
-   * defined as (AC)(CB). The final argument indicates if the coupling is symmetric, in the sense that there
-   * is a two-way coupling between the fields. Without this argument, a nonzero block will be added to the
-   * system matrix for block AB, but block BA will remain zero (one-way coupling).
-   *
-   * - Example 1 = ("node_field","elem_field", ELEM, true) couples all dofs sharing a common element (two-way coupling)
-   * - Example 2 = ("node_field_1","node_field_2", NODE, true) couples all dofs sharing a common node (two-way coupling)
-   * - Example 3 = ("node_field_1","face_field", NODE, false) couples nodal dofs to adjacent faces (one-way coupling)
+   * @copydoc addCoupling(string const &, string const &, Connector, std::vector< string > const &, bool)
    *
-   * @param [in] rowFieldName string the name of the row field.
-   * @param [in] colFieldName string the name of the col field.
-   * @param [in] connectivity Connectivity through what they are connected.
-   * @param [in] regions names of regions where this coupling is defined.
-   * @param [in] symmetric bool is it symmetric, i.e., both row-col and col-row?
+   * Overload for arrayView1d<string> input used by physics solvers.
    */
   void addCoupling( string const & rowFieldName,
                     string const & colFieldName,
-                    Connector const connectivity,
+                    Connector connectivity,
                     arrayView1d< string const > const & regions,
-                    bool const symmetric );
+                    bool symmetric = true );
```

Comment:
Would it be better or worse to not specify a default here?

Reply:
I think `true` is a reasonable default that is used close to 100% of the time. If one way coupling is assumed, don't people tend to do operator splitting instead of a monolithic matrix approach?